### PR TITLE
[tests] Fix flake8 warnings in several wallet functional tests

### PR DESCRIPTION
**Pull request by jnewbery**
Fixes flake8 warnings in several wallet functional tests.

Several wallet functional tests need rewrite to remove the accounts API (#13075). To prepare for that, I fixed all the flake8 warnings in those tests.

#13075 is blocked on a bitcoind bug. This PR is just the flake8 fixes so we're not completely blocked.

### CODE REVIEW

File: test/functional/wallet_basic.py [hunk 1]
```diff
@@ -215,89 +227,89 @@ def run_test(self):
...
-        #3. sign and send
-        #4. check if recipient (node0) can list the zero value tx
+        # check if we can list zero value tx as available coins
+        # 1. create rawtx
+        # 2. hex-changed one output to 0.0
+        # 3. sign and send
+        # 4. check if recipient (node0) can list the zero value tx
         usp = self.nodes[1].listunspent()
-        inputs = [{"txid":usp[0]['txid'], "vout":usp[0]['vout']}]
+        inputs = [{"txid": usp[0]['txid'], "vout":usp[0]['vout']}]
```
Comment (MarcoFalke) [inline test/functional/wallet_basic.py]: If you add a space here, then make it consistently for the second kv as well

File: test/functional/wallet_listtransactions.py [hunk 2]
```diff
@@ -129,21 +136,21 @@ def get_unconfirmed_utxo_entry(node, txid_to_match):
         assert_equal(utxo_to_use["safe"], False)
 
         # Create tx2 using createrawtransaction
-        inputs = [{"txid":utxo_to_use["txid"], "vout":utxo_to_use["vout"]}]
+        inputs = [{"txid": utxo_to_use["txid"], "vout":utxo_to_use["vout"]}]
```
Comment (MarcoFalke) [inline test/functional/wallet_listtransactions.py]: same here

**Comment by MarcoFalke** [comment]
utACK 5ec11f44f322be851746903578bd5224b9807e51. I am not aware of any significant pull request that would conflict due to this, so it seems fine.

**Review by ryanofsky — COMMENTED** [review]
utACK 5ec11f44f322be851746903578bd5224b9807e51

**Comment by jnewbery** [comment]
@MarcoFalke - addressed your comments. I've also fixed a couple of variable names which I somehow missed before.

@ryanofsky - Thanks for the quick review. `git diff 5ec11f4 a533834d` should be minimal

**Comment by MarcoFalke** [comment]
re-utACK a533834d502ef72aa14216828ae0ff5c7fc5625e

**Review by ryanofsky — COMMENTED** [review]
utACK a533834d502ef72aa14216828ae0ff5c7fc5625e

**Comment by MarcoFalke** [comment]
Note that this is no longer [trivial] since you change variable names and remove assignments...

**Pull request merged**